modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.js

```diff
@@ -131,14 +134,18 @@ export default function FieldsSidebarSettingsBody() {
 						}
 						case CORE_EVENT_TYPES.FIELD.EVALUATE:
 							dispatch({
-								payload: {settingsContextPages: payload},
+								payload: {
+									settingsContextPages: payload,
+									submitButtonId,
```

Comment:
Since `submitButtonId` is a configuration you can retrieve this information directly into the reducer.